decrypt/decrypt.go

```diff
@@ -59,6 +59,9 @@ func DataWithFormat(data []byte, format Format) (cleartext []byte, err error) {
 		key,
 		tree.Metadata.LastModified.Format(time.RFC3339),
 	)
+	if err != nil {
```

Comment:
In case `err != nil`, very likely the check `originalMac != mac` also fails, so *an* error would have already been returned - but the message would not be so helpful (probably something like `"...: expected mac xxx, got "`). So handling the error here is definitely a good thing!